zulipterminal/config/keys.py

```diff
@@ -415,6 +415,7 @@ class KeyBinding(TypedDict):
     "msg_actions": "Message actions",
     "stream_list": "Stream list actions",
     "msg_compose": "Composing a Message",
+    "editor": "Editor actions",
```

Comment:
This may change if we split the category, but I'm not sure if editor/input/text/entry are better words. I remember reading about this somewhere on czo semi-recently, but can't seem to dig out a reference.

The web app doesn't have a similar category.

Reply:
I tried searching CZO.
I could only come up with "Editor Navigation" and "Text Operations", on my own.
Should I keep trying or proceed with these for now?

Reply:
Generally if there are other changes to be made and you're unsure of a name or text (or something else that doesn't affect the changes to be made too much), it's easier to make that change in an overall update to the PR with a name you select, and highlight that you're unsure in eg. a self-review within the PR (on the line, if you want) before marking it for review. Then the other changes are already addressed, and it's possible the name you select will be fine - or it can be either updated quickly before merging by a committer, or discussed first.

However, since I already was thinking about it...

Since the name here refers to the same type of component, I often prefer to keep the same prefix, rather than use a phrase that can either have the word that refers to the component move around, or be different. That can then be a phrase like `Prefix using part 2`, or eg. `Prefix - part 1` and `Prefix - part 2`. The latter form may be clearer here, and in terms of the choice of textbox/editor/... itself, we can always update that name later; generally I just aim to be consistent.

The library doesn't support all the features, but https://www.man7.org/linux/man-pages/man3/readline.3.html#EDITING_COMMANDS provides some ideas for naming.

So for the parts we could have
- navigation or movement (based on the man page)
- manipulation? modification?